test/e2e/scheduler_predicates.go

```diff
@@ -297,7 +297,7 @@ var _ = framework.KubeDescribe("SchedulerPredicates [Serial]", func() {
 		framework.ExpectNoError(err)
 		for _, pod := range pods.Items {
 			_, found := nodeToCapacityMap[pod.Spec.NodeName]
-			if found && pod.Status.Phase == api.PodRunning {
+			if found && pod.Status.Phase != api.PodSucceeded && pod.Status.Phase != api.PodFailed {
```

Comment:
It was pending. The problem was that after reboot test static Pods for fluentd and kube-proxy were created, but weren't running before the test started, so it didn't accounted resources allocated for them.